examples/demo-app/index.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset='UTF-8' />
     <title>kepler.gl demo</title>
-    <link href="https://d1a3f4spazzrp4.cloudfront.net/uber-fonts/4.0.0/superfine.css" rel="stylesheet">
-    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css" rel="stylesheet">
+    <link href="http://d1a3f4spazzrp4.cloudfront.net/uber-fonts/4.0.0/superfine.css" rel="stylesheet">
+    <link href="http://api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css" rel="stylesheet">
```

Comment:
You might want to use `//` (protocol-less) since GitHub pages does have `https` support and it can be turned on.

Reply:
See https://uber.github.io/kepler.gl/#/

Reply:
Actually, this would break the demo on GitHub.